app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
```

Comment:
Consider how you could get both of these data points with just one API/facade call (hint: in Ruby you can return more than one item from a method). The benefit here is minimizing API hits